app/parse.py

```diff
@@ -8,8 +15,42 @@ class Quote:
     tags: list[str]
 
 
+def get_all_quotes() -> list[Quote]:
+    all_quotes = []
+    url = BASE_URL
+    while url:
+        response = requests.get(url).content
+        soup = BeautifulSoup(response, "html.parser")
+        quotes_div = soup.find_all("div", class_="quote")
+        for quote_div in quotes_div:
+            text = quote_div.find("span", class_="text").get_text(strip=True)
+            author = quote_div.find(
+                "small",
+                class_="author"
+            ).get_text(strip=True)
```

Comment:
it's easier to use soup.select(".quote") that will find all blocks with class quote